src/server/routes/__tests__/oembedArticleRoute-test.ts

```diff
@@ -16,11 +16,33 @@ const unvalidArticleUrl = "https://test.ndla.no/subject:3";
 const validLearningpathUrl = "https://test.ndla.no/subject:3/topic:1:55163/topic:1:168398/resource:1:9876";
 
 test("oembedArticleRoute success", async () => {
-  nock("http://ndla-api").get("/taxonomy/v1/nodes/urn:resource:1:1682?language=nb").times(2).reply(200, {
-    id: "urn:resource:1",
-    contentUri: "urn:article:123",
-    name: "Resource title",
-  });
+  nock("http://ndla-api")
+    .get("/taxonomy/v1/nodes/urn:resource:1:1682?language=nb")
+    .times(2)
+    .reply(200, {
+      id: "urn:resource:1",
+      contentUri: "urn:article:123",
+      name: "Resource title",
+      breadcrumbs: ["Fag", "Emne"],
+      resourceTypes: [
+        {
+          id: "urn:resourcetype:subjectMaterial",
+          parentId: "urn:resourcetype:tasksAndActivities",
+          name: "Fagstoff",
+          translations: [],
+          supportedLanguages: [],
+          connectionId: "",
+        },
+        {
+          id: "urn:resourcetype:academicArticle",
+          parentId: null,
+          name: "Fagartikkel",
+          translations: [],
+          supportedLanguages: [],
+          connectionId: "",
+        },
+      ],
```

Comment:
Pirk, men det skjærer litt i augene mine at dette ikkje heng sammen.
```suggestion
      resourceTypes: [
        {
          id: "urn:resourcetype:subjectMaterial",
          parentId: null,
          name: "Fagstoff",
          translations: [],
          supportedLanguages: [],
          connectionId: "",
        },
        {
          id: "urn:resourcetype:academicArticle",
          parentId: "urn:resourcetype:subjectMaterial",
          name: "Fagartikkel",
          translations: [],
          supportedLanguages: [],
          connectionId: "",
        },
      ],
```

Reply:
Hehe glemte visst å endre disse